target/linux/ath79/dts/ar9344_engenius_ecb600.dts

```diff
@@ -0,0 +1,212 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9344.dtsi"
```

Comment:
Move ar9344.dtsi before the other includes (see recent changes to all other files in ath79)